src/instructlab/model/evaluate.py

```diff
@@ -269,6 +270,14 @@ def get_model_name(model_path):
     return os.path.basename(os.path.normpath(model_path))
 
 
+def get_cpu_count():
+    try:
+        # Not available on all platforms
+        return len(os.sched_getaffinity(0))  # type: ignore[attr-defined]
+    except AttributeError:
+        return multiprocessing.cpu_count()
```

Comment:
nit: might we want this in the `utils.py` file?

Reply:
Agreed if there needs to be other usage.  The only other place using it now is phased training and it should switch to max_workers="auto" shortly.